src/main/resources/templates/calendar.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+		<title>National Cybersecurity Laboratories - Pricing</title>
+		<object th:include="fragments/header :: header" th:remove="tag" />
+	</head>
+<body>
+
+<!-- main nav -->
+<div class="main-nav navbar-fixed-top">
+	<div class="container">
+		<div class="navbar-header">
+			<a href="index.html" th:href="@{/}"><img src="images/ncl_favicon1.png" alt="logo" width="53" height="53" class="logo1" /></a>
+			<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+				<span class="sr-only">Toggle navigation</span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</button>
+		</div> <!-- navbar-header -->
+		<div class="collapse navbar-collapse">
+			<ul class="nav navbar-nav navbar-left login-navbar-left">
+				<li class="scroll"><a href="about.html">About</a></li>
+				<li class="scroll"><a href="event.html">Events</a></li>
+				<li class="scroll"><a href="plan.html">Features</a></li>
+				<li class="scroll"><a href="pricing.html">Pricing</a></li>
+				<li class="scroll"><a href="research.html">Research</a></li>
+				<li class="dropdown">
+					<a class="dropdown-toggle" data-toggle="dropdown" href="#">Support
+						<span class="caret"></span></a>
+					<ul class="dropdown-menu ">
+						<li><a href="http://ncl.sg/docs-ncl/">Tutorial</a></li>
+						<li><a href="resources.html">External Resources</a></li>
+						<li><a href="calendar.html">Usage Calendar</a></li>
+					</ul>
+				</li>
+				<li class="scroll"><a href="contactus.html">Contact Us</a></li>
+			</ul>
+			<ul class="nav navbar-nav navbar-right login-navbar-right">
+				<li><a href="#" th:href="@{/signup2}">Create account</a></li>
+			</ul>
+			<ul class="nav navbar-nav navbar-right login-navbar-right1">
+				<li><a href="#" th:href="@{/login}">Log In</a></li>
+			</ul>
+		</div> <!-- navigation links -->
+	</div>
+</div> <!-- main-nav -->
+    <p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+	<p>&nbsp;</p>
+
+	<div align="center">
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "align" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)